res/scripts/script-index.js

```diff
@@ -1 +1,28 @@
-console.log("Test");
+//AddEventListener to DOMContentLoaded, so the image waits to load
+//after the page is loaded
+document.addEventListener("DOMContentLoaded", function(event) {
+ // onload_image();
+  createDiv();
+ 
+});
+
+  //Function to create the div elements for the grid
+  function createDiv() {
+    var parentDiv = document.getElementById("container");
+    for(i=1; i<=36; i++){
```

Comment:
Careful with global variables.
```suggestion
    for(let i=1; i<=36; i++){
```
http://tobyho.com/2011/10/25/js-accidental-global-variables/

Reply:
Maybe consider renaming also i to something better, like index.

Reply:
Fixed :)